dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,193 @@
+{
+    "pages": [
+        {
+            "title": "Landing Page",
+            "url": "/",
+            "sections": [
+                {
+                    "title": "Welcome to Dataland",
+                    "text": [
+                        "This is the main introduction of our homepage."
+                    ],
+                    "image": [
+                        "/static/logos/gfx_logo_dataland_orange_S.svg"
+                    ]
+                },
+                {
+                    "title": "Intro",
+                    "text": [
+                        "Liberate Data - ",
+                        "Empower Autonomy.  ",
+                        "The alternative to data monopolies."
+                    ],
+                    "image": [
+                        "/static/logos/Dataland_D_svg_Logo.svg"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "/static/icons/Search.svg",
+                            "text": "Search for a company to view or request data..."
+                        }
+                    ]
+                },
+                {
+                    "title": "Quotes",
+                    "text": [
+                        "START YOUR DATALAND JOURNEY"
+                    ],
+                    "cards": [
```

Comment:
Question: Will there be two more Youtube videos in the quotes?  I remember having heard that there will be three videos at the end. Is that correct?

Reply:
Yes, three video in total for now. I got them yesterday and gonna work on it today